File: 1__C_Program/3__area_perimeter.c
#include <stdio.h>

int main() {
    // Declare variables and input length, width
    float length, width;
    printf("Enter length of a rectangle: "); scanf("%f", &length); 
    printf("Enter width of a rectangle: "); scanf("%f", &width);

    // Calculate the area and perimeter
    printf("Area of a rectangle is: %.2f\n", length * width);
    printf("Perimeter of a rectangle is: %.2f\n", 2*(length + width));
    return 0;
}

File: 5__JavaScript_Program/3__area_perimeter.js
const prompt = require('./node_modules/prompt-sync')({ sigint: true });

// Input width, length and print area, perimeter
const width = Number(prompt("Enter width of a rectangle: "));
const length = Number(prompt("Enter length of a rectangle: "));

const area = width * length;
const perimeter = 2*(width + length);

console.log(`"The area of a rectangle is: ${area}`);
console.log(`"The perimeter of a rectangle is: ${perimeter}`);